common/src/main/java/com/wynntils/models/items/annotators/game/MiscAnnotator.java

```diff
@@ -40,6 +41,6 @@ public ItemAnnotation getAnnotation(ItemStack itemStack, CodedString name) {
         // e.g. a GUI element
         if (!untradable && !questItem) return null;
 
-        return new MiscItem(name, untradable, questItem);
+        return new MiscItem(CodedString.fromStyledText(name), untradable, questItem);
```

Comment:
Was it too much fallout to change MiscItem as well?

Reply:
I'm not sure what you mean by resolving this? Let me phrase it like this:
I think we should change MiscItem so the first argument to the constructor (and the field and getters) are StyledText, instead of CodedString.

And I would like to see that change in this PR, if it is possible.

Reply:
Or, sure, we can fix it later.